### PR TITLE
[UTOPIA-982] - 2 different UI styles for radio buttons

### PR DESCRIPTION
**Pull request by NoorChasib**
# Description

This PR includes the following proposed change(s):

- utilized Radio component to render radio buttons in forms
- forced font weight and margin top to radio buttons
- made checkboxes same color as radio buttons for consistency

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Refactoring / Documentation
- [ ] Updating Testing Framework(s)
- [ ] Version change

## How Has This Been Tested?

locally tested

## Development Dependency Working Agreement
- [x] My code DOES NOT include the importing of new dependencies into the DPIA ecosystem
- [ ] My code DOES include the importing of new dependencies into the DPIA ecosystem
**If new dependencies are being introduced to the DPIA ecosystem:**
- [ ] The functionality of the dependency drastically reduces code complexity and makes my changes more easily maintainable and readable 
- [ ] The dependency being introduced does not contain multiple layers of nested dependencies introducing maintainability complexity to the DPIA ecosystem

## Frontend Development Changes
- [ ] N/A
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to project documentation or diagrams that reflect my changes
- [x] New and existing unit tests pass locally with my changes
- [ ] My code follows Airbnb React Style Guidelines

## API Development Changes
- [x] N/A
- [ ] I have performed a self-review of my own code
- [ ] My code follows standards and practices outlined in the BC Government API Development Guidelines
- [ ] New and existing unit tests pass locally with my changes
- [ ] My changes includes Swagger documentation updates that reflect the changes I am introducing

## Definition of Done

Definition of Done

### CODE REVIEW

File: src/frontend/src/components/public/PIAFormTabs/accuracyRetention/index.tsx [hunk 1]
```diff
@@ -58,6 +59,140 @@ export const AccuracyCorrectionAndRetention = () => {
     setNestedReactState(setAccuracyCorrectionAndRetentionForm, path, value);
   };
 
+  const haveProcessinPlace = [
+    {
+      index: 1,
```
Comment (antsand) [inline src/frontend/src/components/public/PIAFormTabs/accuracyRetention/index.tsx:64]: Hard coding the index can cause issue in-case we want to reshuffle, add. delete an array. The loop will loop through this sequentially so index might be redundant.

**Review by antsand — APPROVED** [review]
Looks super good! Love it!!!

File: src/frontend/src/components/public/PIAFormTabs/accuracyRetention/index.tsx (hunk 1 above)
Comment (NoorChasib) [reply src/frontend/src/components/public/PIAFormTabs/accuracyRetention/index.tsx:64]: I see that now, I can take hardcoded index number out in that case

Comment (NoorChasib) [reply src/frontend/src/components/public/PIAFormTabs/accuracyRetention/index.tsx:64]: Typescipt gives me an error that I'm missing the index. I might just leave it as is instead

**Pull request merged**